v1/examples/basics/core/Footer.js

```diff
@@ -10,7 +10,9 @@ const React = require('react');
 class Footer extends React.Component {
   docUrl(doc, language) {
     const baseUrl = this.props.config.baseUrl;
-    return `${baseUrl}docs/${language ? `${language}/` : ''}${doc}`;
+    const docsUrl = this.props.config.docsUrl;
```

Comment:
Does this work even when `docsUrl` is null ? In which it should have fallbacked to `docs`